ci/prepare.yml

```diff
@@ -0,0 +1,37 @@
+---
+- name: "Prepare the environment for running stf"
+  hosts: controller
+  tasks:
+    - name: "Update pip"
+      ansible.builtin.pip:
+        name: pip
+        state: latest
+        extra_args: "-U"
+
+    - name: "Install pre-reqs from pip"
+      ansible.builtin.pip:
+        requirements: "../build/stf-run-ci/requirements.txt"
+        chdir: "{{ playbook_dir }}"
+        state: present
+
+    - name: "Install ansible collections"
+      community.general.ansible_galaxy_install:
+        type: collection
+        name: "{{ item }}"
+      with_items:
+        - "kubernetes.core:2.3.2"
+        - "community.general:6.2.0"
+
+    - name: "Log into the cluster"
+      ansible.builtin.import_role:
+        name: rhol_crc
+        tasks_from: add_crc_creds.yml
+
+    - name: "Create the service-telemetry project"
+      kubernetes.core.k8s:
+        api_version: v1
+        kind: Namespace
+        name: "service-telemetry"
```

Comment:
Our current upstream ends up creating different namespaces base on PR name etc which at least helps validate that everything can work outside of a statically named project. I think it would be useful to keep that going because I've seen it catch at least one issue in the past.

Reply:
Seems there is a `{{ namespace }}` variable that could be used here at the very least